.github/workflows/push.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+setup_git() {
+  git config --global user.email "ghactions@github.com"
+  git config --global user.name "GitHub Actions"
+}
+
+commit_website_files() {
+  sudo touch .nojekyll
+  git add -A . 
+  git commit --message "GitHub Actions Build: $GITHUB_RUN_ID [ci skip]"
+  git branch temp-changes
+  git checkout $GITHUB_REF_SLUG
+  git merge --no-commit --no-ff temp-changes
+  git reset HEAD -- package.json
+  git reset HEAD -- package-lock.json
+  git commit --message "GitHub Actions Build: $GITHUB_RUN_ID [ci skip]"
+}
+
+upload_files() {
+  git remote add origin-pages https://${GH_TOKEN}@github.com/${GITHUB_REPOSITORY}.git > /dev/null 2>&1
```

Comment:
Do you know if `GH_TOKEN` is available? Or is that a Travis specific thing?

Reply:
`GH_TOKEN` must be pre-defined. It is an ENV setting in travis.
For my tests I've defined it as a GitHub project secret.

https://github.com/opendevstack/ods-documentation/pull/62/files#diff-edd7e4675f0f52ec8d2f51cb50523e3e4cecfcf42b94f3b314d15c0a5d283d2cR28

Reply:
Ah okay cool. Did you look into `GITHUB_TOKEN`, as described in https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow? Just curious, happy to merge this as is and improve later ...

Reply:
I wasn't aware of GITHUB_TOKEN, but changed to it as actions are skipped on push (built-in).